extensions/wikidata/src/org/openrefine/wikidata/manifests/ManifestV1.java

```diff
@@ -25,6 +28,9 @@ public ManifestV1(JsonNode manifest) {
         JsonNode mediawiki = manifest.path("mediawiki");
         name = mediawiki.path("name").textValue();
         mediaWikiApiEndpoint = mediawiki.path("api").textValue();
+        if (mediaWikiApiEndpoint == null || mediaWikiApiEndpoint.isEmpty()) {
```

Comment:
If this happens, it means that the manifest provided is invalid, so perhaps we should throw some Exception? This is kind of a validation to the manifest, it could be better to do this in a fix to #3109.

Reply:
Ah good point, I'll remove it.